ansible/roles/games/tasks/games.yml

```diff
@@ -0,0 +1,10 @@
+- name: pacman install steam and lutris
+  pacman:
+    name:
+    - steam
+    - lutris
+
+- name: install heroic launcer
```

Comment:
I'd like to keep aur dependencies to a minimum, especially since most game installations could be handeled using lutris or steam proton.

Reply:
Keep the rest of gaming?